conanfile.py

```diff
@@ -2,25 +2,24 @@
 
 class libiconvConan(ConanFile):
     name = "libiconv"
-    version = "1.15.0"
-    branch = "master"
+    lib_version = "1.15.0"
+    revision = "0"
+    version = '{}-{}'.format(lib_version, revision)
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    description = "Convert text to and from Unicode"
```

Comment:
[minor] Please be consistent with `"` and `'`. You should use single quotes `'` except if double quotes are really necessary. This applies to the whole file.

Reply:
done